utilities/xmlrpcpp/src/XmlRpcServer.cpp

```diff
@@ -32,14 +56,14 @@ XmlRpcServer::~XmlRpcServer()
 
 
 // Add a command to the RPC server
-void 
+void
```

Comment:
Please revert these whitespace changes (a follow-on commit which re-adds them is acceptable, as it will get squashed down anyway).

Reply:
Reverted.